src/components/Book.jsx

```diff
@@ -9,14 +9,14 @@ const Book = () => {
   const dispatch = useDispatch();
 
```

Comment:
- [ ] Redux store should be passed as per project requirements i.e. Pass your configured store into the <Provider>
```
  <Provider store={store}>
    <App />
  </Provider>,
```

Reply:
Yes, that's what I did. Did you check the link above? [Here](https://github.com/wacoo/react-book-store/blob/630d9877180a88ef3679d9f80dbf23140375d785/src/index.js#L11C5-L13C16)